test/quantization/core/test_docs.py

```diff
@@ -41,28 +41,17 @@ def _get_code(
         def get_correct_path(path_from_pytorch):
             r"""
             Current working directory when CI is running test seems to vary, this function
-            looks for docs and if it finds it looks for the path to the
-            file and if the file exists returns that path, otherwise keeps looking. Will
-            only work if cwd contains pytorch or docs or a parent contains docs.
+            looks for docs relative to this test file.
             """
-            # get cwd
-            cur_dir_path = Path(".").resolve()
-
-            # check if cwd contains pytorch, use that if it does
-            if (cur_dir_path / "pytorch").is_dir():
-                cur_dir_path = (cur_dir_path / "pytorch").resolve()
-
-            # need to find the file, so we check current directory
-            # and all parent directories to see if the path leads to it
-            check_dir = cur_dir_path
-            while not check_dir == check_dir.parent:
-                file_path = (check_dir / path_from_pytorch).resolve()
-                if file_path.is_file():
-                    return file_path
-                check_dir = check_dir.parent.resolve()
-
-            # no longer passing when file not found
-            raise FileNotFoundError("could not find {}".format(path_from_pytorch))
+            core_dir = Path(__file__).parent
+            assert core_dir.match("test/quantization/core/"), (
```

Comment:
Do you want this assert to also fail if `test_docs.py` is moved to a subdirectory of `test/quantization/core/`?